engine/lib/membrane_rtc_engine/track.ex

```diff
@@ -57,6 +60,7 @@ defmodule Membrane.RTC.Engine.Track do
   * `active?` - indicates whether track is still available or not (because client left a room)
   * `metadata` - any data passed by user to be linked with this track
   * `ctx` - any data Endpoints need to associate with `#{inspect(__MODULE__)}.t()` for internal usage
+  * `framerate` - framerate of video track used only when a track is read from a file. By default nil
```

Comment:
We shouldn't refer to specific endpoints in the engine docs. In fact, any endpoint can use this field and we don't want to update engine docs when it happens. Also, anyone can write their own endpoint that will make use of this field